src/windows/windows.js

```diff
@@ -134,3 +137,16 @@ exports.setConfigState = data => {
     exports.createTimerWindow()
   }
 }
+
+function createBrowserWindowOnCurrentScreen (options) {

```

Comment:
This method name doesn't quite describe what's really going on here. Without the context of the full-screen issue, I'd not understand why this is different from just making a new `BrowserWindow` in the first place.

Maybe something more like `createBrowserWindowAboveFullScreenApps` or something like that?

Reply:
Went with `createAlwaysOnTopBrowserWindow`